openapi/paths/storefront/kyc-liveness-sessions@{id}@finish.yaml

```
@@ -8,12 +8,16 @@ post:
   summary: Finish KYC liveness session
   operationId: StorefrontPostKycLivenessSessionFinish
   x-sdk-operation-name: finish
-  description: Attempt to finalize a pending KYC liveness session.
+  description: |-
+    Attempts to finalize a pending KYC liveness session.
+    The session can only be finalized when the KYC liveness session, in the third-party service, is in a final state.
```

```suggestion
    The session can only be finalized when the KYC liveness session in the third-party service, is in a final state.
```
Not sure seems unnecessay comma here

---

I thought it might be good to break up the sentence a little. It seems quite long. I don't have strong feelings on it though

---

I agree with @noon-dawg to be honest. Otherwise it's a little too much of a run-on sentence.

---

```suggestion
```

I think we should remove it. It sounds too much like the transaction can be marked as completed when the gateway completed processing.